tests/test_transform/factory/test_correctness.py

```diff
@@ -19,10 +19,18 @@
     TransformFactory,
     TransformScheme,
 )
-from compressed_tensors.utils import align_modules, force_cpu_offload
+from compressed_tensors.utils import force_cpu_offload
 from tests.testing_utils import requires_accelerate, requires_gpu
 
 
+_test_schemes = [
```

Comment:
do we want these as `@pytest.fixture`s?

Reply:
Pytest fixtures can't actually be used for parameterization afaict (or without annoying yielding logic). I have cleaned this up to be a little more readable though.